pkg/logs/message/origin.go

```diff
@@ -49,18 +49,15 @@ func (o *Origin) TagsPayload() []byte {
 	if o.LogSource.Config.SourceCategory != "" {
 		tagsPayload = append(tagsPayload, []byte("[dd ddsourcecategory=\""+o.LogSource.Config.SourceCategory+"\"]")...)
 	}
-	var tags string
-	if o.LogSource.Config.Tags != "" {
+	var tags []string
+	if len(o.LogSource.Config.Tags) != 0 {
```

Comment:
nit: could we stick to `> 0` for consistence?

Reply:
Done